Makefile

```diff
@@ -60,7 +60,7 @@ install:
 	$(GO) install
 
 clean:
-	$(GO) clean
+	$(GO) clean -i . ./cli ./core ./mocks ./testaid ./util
```

Comment:
maybe add a comment to explain why we need to supply all directories now?

Reply:
It wasn't just adding all directories. The -i option is crucial as well.  Comment added.